manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: WatchGuard.MobileVPNwithSSLclient

```

```suggestion
PackageIdentifier: WatchGuard.MobileVPNWithSSLClient
```
PackageId is generally written in "Pascal" case